django_ledger/forms/data_import.py

```diff
@@ -13,7 +13,8 @@ class ImportJobModelCreateForm(ModelForm):
         label='Select File...',
         widget=forms.FileInput(
             attrs={
-                'class': 'file-input'
+                'class': 'file-input',
+                'accept': '.ofx'
```

Comment:
I think it is a good idea to limit the file extension. However, QFX is also a popular OFX compatible format, please modify this line to accept both, OFX and QFX files as follows:

```py
widget=forms.FileInput(
            attrs={
                'class': 'file-input'
                'class': 'file-input',
                'accept': '.ofx,.qfx'
})
```

Thank you for your contribution to the project!

Reply:
This is added.